src/node/desktop/package.json

```diff
@@ -36,7 +36,6 @@
     "test": "electron-mocha -c",
     "testcover": "nyc electron-mocha -c",
     "testwip": "electron-mocha -c -g WIP",
-    "use-release": "rm -rf src/config && ln -nfs ../config/release src/config",
-    "use-dev":     "rm -rf src/config && ln -nfs ../config/development src/config"
+    "use": "node ./scripts/use.js"
```

Comment:
How do we "use" this now (no pun intended, mostly)? `yarn use development` or something like that?

Gotta be a better way to do this sort of thing, hopefully it will come to us in a dream.